modules/oe_showcase_news/config/install/core.entity_view_display.node.oe_news_simple.default.yml

```
@@ -0,0 +1,51 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_news_simple.field_news_content
+    - field.field.node.oe_news_simple.field_news_image
+    - field.field.node.oe_news_simple.field_news_publication_date
+    - node.type.oe_news_simple
+  module:
+    - datetime
+    - image
+    - text
+    - user
+id: node.oe_news_simple.default
+targetEntityType: node
+bundle: oe_news_simple
+mode: default
+content:
+  field_news_content:
+    weight: 3
+    label: above
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  field_news_image:
+    weight: 1
+    label: above
+    settings:
+      image_style: ''
+      image_link: ''
+    third_party_settings: {  }
+    type: image
+    region: content
+  field_news_publication_date:
+    weight: 2
+    label: above
+    settings:
+      timezone_override: Europe/Brussels
+      format_type: news_full_time_format
```

use short till we have design 
```suggestion
      format_type: short
```